examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -178,11 +186,8 @@ async function getAllElementsByLocator(
   return await locatorResult.all();
 }
 
-async function enterEmailAndContinue(
-  page: Page,
-  userId: string,
-): Promise<void> {
-  const email = `${userId}@revenuecat.com`;
```

Comment:
The user ID was only used to generate an email. I replaced it by a hardcoded email since we don't know the app user ID in the test when it's autogenerated.